jujugui/static/gui/src/app/app.js

```diff
@@ -2903,6 +2903,10 @@ YUI.add('juju-gui', function(Y) {
       if (!gui || (gui && !gui.inspector)) {
         this._renderAddedServices();
       }
+
+      if (this.get('gisf')) {
```

Comment:
This method gets called every time the db updates - We don't need to render this every time the db updates.